src/components/MyProfile/MyProfile.jsx

```diff
@@ -0,0 +1,59 @@
+import ListGroup from 'react-bootstrap/ListGroup';
+import { useSelector } from 'react-redux';
+import Stack from 'react-bootstrap/Stack';
+
+function MyProfile() {
+  // Rockets State
+  const rockets = useSelector((state) => state.rockets);
+  // Missions State
+  const missions = useSelector((state) => state.missions);
+  // Dragons State
+  const dragons = useSelector((state) => state.dragons);
+  // Reseved dragons
+  const reservedDragons = dragons.filter((dragon) => dragon.reserved === true);
+  // Joined Missions
+  const joinedMission = missions.filter((mission) => mission.reserved === true);
+  return (
+    <Stack className="container" style={{ border: 'none' }} direction="horizontal" gap={3}>
+      {/* Display Rockets */}
+      <div className="row">
+        <div className='col-6'>
```

Comment:
- Let's kindly fix the issue with the use of single quotes in the class attribute on the div with the class of `col-6`. Update it to this:

` <div className="col-6">`